src/flatbuffers/flatbuffers_utils.h

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "../flatbuffers.h"
+#include "../span.h"
+
+#include "schema/genai_lora.fbs.h"
+
+#include <string>
+#include <string_view>
+#include <unordered_map>
+
+struct OrtValue;
+
+namespace Generators {
+namespace lora_parameters {
+namespace utils {
```

Comment:
Can we use Pacal casing for namespace to be consistent with the project namespace?

Reply:
I am actually against pascal casing for namespaces. `Generators` is actually an outlier in all of the surrounding code. Propose to fix that in a separate PR.